Makefile

```diff
@@ -55,6 +55,11 @@ local-run: local-build
 	@echo "Running local app container"
 	@docker-compose -f docker-compose.dev.yml up
 
+local-build-run:
```

Comment:
Isn't this similar to what `local-run` does already? It calls `local-build`

Reply:
Yes, I didn't notice that. This become redundant. But still this is a single cmd which is building and running.  I will remove this.